ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
```

Comment:
See, I'm not sure it's clear enough for what it is. "GUI responsiveness / FPS improvements / Lag reduction" would I think be better, it's been such a problem for some players (including me) for so long that it needs to be clear that we've substantially reduced it. Given I didn't understand that that point was saying that, it won't just be me that misses it.